File: Acceleration/src/Circle.java
import processing.core.PApplet;
import processing.core.PVector;

public class Circle {
    private PApplet p;
    private PVector position;
    private PVector speed;
    private PVector acceleration;
    private float radius;
    private float maxSpeed;
    private float maxAcceleration;
    private float deltaAcceleration;

    public Circle(PApplet p) {
        this.p = p;
        radius = 10;
        position = new PVector(p.random(radius, p.width - radius), p.random(radius, p.height - radius));
        speed = new PVector(p.random(-5, 5), p.random(-5, 5));
        acceleration = new PVector(0, 0);
        maxSpeed = 8;
        maxAcceleration = 0.5f;
        deltaAcceleration = 0.2f;
    }
}

File: docs/Acceleration/circle.js
class Circle {

    constructor() {
        this.radius = 5;
        this.postion = createVector(random(this.radius, width - radius), random(this.radius, height - radius));
        this.speed = createVector(random(-5, 5), random(-5, 5));
        this.acceleration = createVector(0, 0);
        this.color = new Color(color(0, 0, 255));
        this.maxSpeed = 8;
        this.maxAcceleration = 0.5;
        this.deltaAcceleration = 0.2;
    }
}
